### ISSUE TITLE
Consider differentiating Human Laser and missle weapons damage profiles for more interesting tactical possibilities

### ISSUE THREAD

**Issue by Pointedstick** [endless-sky/endless-sky#3266]
I've [brought this up before](https://github.com/endless-sky/endless-sky/issues/183), but I wanted to take the opportunity to do so once more in an integrated manner, reflecting years of gameplay and observation of player trends.

## The issue ##
Human beam weapons favor hull over shield damage, likely for balance reasons, because they're better in all other ways than other Tier 1 human weapons. But this is sort of backwards. Look at conventional military lasers today: they're focused, not indiscriminate; they burn small holes after several seconds of firing--it's nothing compared to the damage done by a missile or machine gun (as Sunbeams reflect!). And besides, it's boring for Lasers and other beam weapons to be so dominant in popularity vs projectile weapons.

## The proposal ##
- Make Human beam weapons favor shield damage over hull damage--maybe a little more shield damage and much less hull damage
- Make missile weapons (especially Human ones) strongly favor hull damage over shield damage--much more hull damage and a bit less shield damage
- Keep non-explosive projectile/gun weapons fairly balanced vis-a-vis balance of shield vs hull damage

## Why it would be a gameplay improvement ##
Changing Human Beam weapons' damage profiles to favor shield damage wouldn't be overpowering given the other changes proposed in https://github.com/endless-sky/endless-sky/issues/3264 and https://github.com/endless-sky/endless-sky/issues/3265 that would make other gun options and small turrets more competitive. Strongly differentiating missiles and lasers' damage profiles would let them complement one another and encourage players who favor Lasers to employ missile weapons as well, and reward them for using the right tool for the job. Players preferring take-all-comers weapons would be encouraged to favor Energy Blasters, Proton Guns, Particle Cannons, Pulse Cannons, Thrasher Cannons, Repeaters, and Bombardment Cannons over Lasers, Electron Beams, Sunbeams, and Disruptors, which are currently boringly and excessively popular. And when faced with heavily-armored Korath drones, players might rationally decide to go back to Human Heavy Rockets and Typhoon Torpedoes (especially against the Mereti, which lack anti-missile weaponry). All of this would offer more practical and realistic options for ship loadouts, which would make the game more interesting!

**Comment by ghoulavenger** [comment]
I favor beam weapons for three major reasons.

The first reason is that they stop firing when you disable a ship. This makes it significantly easier to board ships. This reason dies down a little bit when you get into fleet combat because of crossfire, but it still beats the alternative.

The second reason is that beam weapons are effective at close quarters. Projectile weapons tend to be good if you want to run and gun rather than fight a close quarters fight.

The last reason is that secondary weapons (which you're trying to push) cost either time or money to replenish. Which means you doing this is trying to force me to use a weapon that I have no interest in, not because it isn't valuable, but because it adds unnecessary expense to my fleet -- I can get more value out of using that money to buy a new ship and throwing more weapons on it.

So you haven't addressed two of the primary reasons I use beam weapons:
1. They give me the greatest chances of boarding vessels.
2. They don't cost money.

You did somewhat address one of them:
1. They are far more practical in close quarters

But even here you didn't really address my reasoning directly. If beam weapons are geared as versus shields primarily the only thing you have done is try to force a secondary weapon on me rather than reduce their effectiveness. It isn't as if the secondary weapon can replace it since you're suggesting it be the exact opposite of the beam weapon, which in my opinion is actually a nerf to secondary weapons, where range is one of their great advantages (which you want to weaken by forcing them into a combination with short range weapons) -- excepting perhaps the torpedo and typhoon which have poor tracking iirc (so are probably used close ranged anyway).

**Comment by Pointedstick** [comment]
I strongly support boosting secondary weapon effectiveness too, but that's another issue. :slightly_smiling_face:

**Comment by Amazinite** [comment]
I think I have to entirely disagree in this regard. I've done a few breakdowns (https://github.com/endless-sky/endless-sky/pull/3259#issuecomment-346405771 and https://github.com/endless-sky/endless-sky/issues/2662#issuecomment-310805362) on how a weapon's shield to hull damage ratio affects its effectiveness against most ships in the game, who in almost all cases have more shields than hull (https://github.com/endless-sky/endless-sky/issues/2662#issuecomment-310722132). Because of this, a weapon that has lower overall damage than another weapon might actually kill ships faster given that the weaker looking weapon does more shield damage than hull damage (much more, in the ball park of 2.4 for your average encounter) and the stronger looking weapon does more hull damage than shield damage (the further the shield/hull damage ratio of the weapon is from that of the ship's hit points, the longer it takes to kill).

Currently:
* Human lasers: ~0.75 shield/hull damage ratio on average (hull > shield)
* Human projectile weapons: ~1.5 shield/hull damage ratio on average (shield > hull)
* Human secondary weapons: ~1.2 shield/hull damage ratio on average (shield > hull, but closer to balanced)
  * It's important to note that within the secondary weapons, the Gatling Gun has a 0.5 shield/hull ratio (very must hull > shield), Typhoon, Torpedo, and Heavy Rocket launchers are close to balanced (with the former two being exactly balanced between shield and hull), and the missile weapons (Javelin, Meteor, Sidewinder) being more shield heavy in their damage (with the Javelin being very much shield > hull at a ratio of 1.85).

Your proposal then is to flip lasers from hull heavy to shield heavy (increasing their effectiveness against almost EVERY ship in the game), flatten out projectile weapons, and make secondary weapons more hull-heavy like the Gatling gun. Personally, I only see this ruining variety. We already have "interesting tactical possibilities" in the game with the weapons that we have. The plasma cannon is an immensely shield heavy weapon (2.52 shield damage for 1 hull damage) while lasers are much more hull heavy. Human secondary weapons currently have a wide variety of shield/hull ratios among them, with some being hull heavy, some being balanced, and some being shield heavy. Human projectile weapons also have some variety, with particle and proton cannons being very balanced, blasters being moderately shield heavy, and plasma cannons being very shield heavy in damage. Within the categories of laser, projectile, and secondary weapon, the only class without any variety is lasers, but that variety is found later down the road where Sunbeams are the exact opposite of human lasers, doing more damage to shields than hull.

Without tweaking numbers, human laser weapons become more powerful than they already are, secondary weapons in some ways become even more useless than many may consider them right now, and projectile weapons become generally bland.

The only reason I could see implementing these changes would be to aim for a sense of "realism," but without seeing exactly how you would tweak the numbers yourself, I can't agree that this would create "more interesting tactical possibilities."

**tl;dr**
>Strongly differentiating missiles and lasers' damage profiles would let them complement one another and encourage players who favor Lasers to employ missile weapons as well, and reward them for using the right tool for the job.

**Certain weapons will already compliment each other, it's just that entire classes of weapons are not guaranteed to compliment a weapon of a different class. And this...**

>Lasers, Electron Beams, Sunbeams, and Disruptors, which are currently boringly and excessively popular.

**is more a matter of the fact that laser weapons make disabling-and-not-destroying far easier, something that players desire due to the lucrativeness of capturing ships.**

**Comment by Pointedstick** [comment]
Yeah, I agree that the cannons need a buff compared to lasers. I can't argue with anything else you wrote, and I admit this is probably the more controversial of the proposals. It just seems silly that Lasers are so much better in so many ways.

**Comment by Amazinite** [comment]
Plasma Cannons and Electron Beams actually compliment each other extremely well, but if you're only using one type of weapon then it's lasers all the way right now, and I don't think that changing the shield/hull ratios of weapons around would change that as long as their damage/tons don't change as well, and since the damage/ton is where the real problem is at, I wouldn't touch the shield/hull ratios.

**Comment by Amazinite** [comment]
Heck, this is making me want to open an issue on the balance of Particle Cannons, Plasma Cannons, and Proton Guns, as right now they're some of the weakest guns per-ton in the game, and all the other weapons that are generally as weak have special damage types to them (e.g. Detainer, Ion Cannon, Ion Rain Gun, Inhibitor). The Plasma Cannon is probably fine because as far as actual combat goes, the heat damage gives it a huge bonus over an Electron Beam, but the Proton Gun and Particle Cannon just aren't there in terms of competing with lasers.

**Comment by Pointedstick** [comment]
The Particle Cannon at least has range and pushback, which are nice. But the Proton cannon just stinks. I think it should be 20 tons max so you could stuff two of 'em into fighters.

**Comment by KiLEdEnNis** [comment]
in short & simple words (since this thread is half an essay already):
laser weapons have high accuracy as an advantage while projectile weapons have to "predict" due to their travel time, so projectile weapons should have generally higher damage (more "risk-reward"sih) while lasers are steady ensured DPS

**Comment by ghoulavenger** [comment]
@KiLEdEnNis accuracy is only a limiter on the added range of projectile weapons. In close quarters, projectiles will almost always hit making them very little different than beam weapons in consistency (it's the difference between shooting the broad side of a barn and shooting the head of a pin). If there is any lost bullets in close quarters I'd argue that it has to do with using forward mounted guns rather than turrets (and not all projectiles can be mounted on turrets sadly). And yes, I do use projectile weapons, just usually in the early game, and especially on cargo fleets.

**Comment by SirStigII** [comment]
What about changing lasers to needing a cool down period, most IRL lasers don't run continuously, they do a burst and then have a cool down period. This behaviour might be too similar to Korath repeaters, but I think it might add a bit of a deterrent to using lasers, as a fighter could potentially be missed entirely if the turret had just fired. That and sticking all the other projectile guns into turrets, as I can't see how Lovelace Labs hasn't seen the dollar signs that would come from single and dual turreted particle cannons, and a proton turreted dreadnaught would serve as a giant AA battery.

**Comment by Bladewood** [comment]
Yes, it's too similar.
In fact, it basically makes them tier 1 slicers. Or moonbeams. Which are already very similar to slicers.

**Comment by Pointedstick** [comment]
Nah.

**Comment by Pointedstick** [comment]
Proton gun improvements: https://github.com/endless-sky/endless-sky/issues/3287
